Angular/src/app/other_components/expandable-text/expandable-text.component.ts

```diff
@@ -0,0 +1,87 @@
+import { Component, Input, ElementRef, AfterViewInit, ChangeDetectorRef } from '@angular/core';
+//import { CommentaryComponent } from '../../commentary/commentary.component';
+
+@Component({
+  selector: 'app-expandable-text',
+  templateUrl: './expandable-text.component.html',
+  styleUrls: ['./expandable-text.component.scss'],
+})
+export class ExpandableTextComponent implements AfterViewInit {
+  @Input() content: string; // HTML content
+  @Input() isCollapsed?: boolean;
```

Comment:
isCollapsed and collapsedHeightPx are not used as input right?

Reply:
In dit geval zijn ze niet gebruikt. maar je zou deze properties in een latere use-case kunnen gebruiken als extra instellingen. beetje de vraag of isCollapsed als default false meegeven nuttig is, maar de collapsedHeightPx configuarable maken leeg me iig wel nuttig.

Reply:
collapsed height zou je dan misschien in de env willen zetten zodat het heel makkelijk aan te passen is voor de programmeur. maar in principe staat ie gelijk onder de constructor precies waar ie moet staan: hij is verder toch voor de user niet configurable. Ik zou de inputs gewoon weghalen aangezien ze niet gebruikt worden; voegen we ze gewoon toe als we ze nodig hebben :D